src/tests/functional/plugin/shared/src/main.cpp

```diff
@@ -3,12 +3,30 @@
 //
 
 #include "gtest/gtest.h"
+#include "gflags/gflags.h"
 
 #include "functional_test_utils/summary/environment.hpp"
 #include "functional_test_utils/summary/op_summary.hpp"
 #include "functional_test_utils/skip_tests_config.hpp"
 
+DEFINE_int32(gpu_device_id, -1, "GPU Device ID (a number starts from 0)");
+
 int main(int argc, char *argv[]) {
+    printf("Running main() from %s\n", __FILE__);
+
+    GFLAGS_NAMESPACE::AllowCommandLineReparsing();
+    gflags::ParseCommandLineFlags(&argc, &argv, false);
```

Comment:
We have another cmd line options for this test binary which don't use gflags, so I think we need to have aligned parsing. 
@iefode Should we use gflags for all options or custom parsing for device ID?

Reply:
I think no. We don't use `gflags` here. Please use the previous code style. I am not sure that it's the best option to specify device as command line option here.
@vladimir-paramuzov Lets discuss offline

Reply:
Done.